plugins/Number.py

```diff
@@ -46,11 +46,27 @@ def init(self, logger):
 '''Check that the value is accurate. Consider whether another tag should
 be used if the value is valid.''')
         )
+        self.errors[3093] = self.def_class(item = 3091, level = 2, tags = ['value', 'fix:chair'],
+            title = T_('Suspicious value'),
+            detail = T_(
+'''The tag expects a positive, round number without unit'''),
```

Comment:
```suggestion
'''The tag expects a positive, round number without unit.'''),
```

Reply:
This one was actually already fixed in the 3th commit :)